reports/Google-2018-report.md

```diff
@@ -13,11 +15,10 @@ prioritize transparency, diversity, and reproducibility to foster collaboration.
 Imagine you are a carrot farmer trying to make dinner. You could make carrot soup by yourself, but you could also share ingredients 
 with the neighbours to make a much tastier soup with onions, greens, and some salt. Some people may be able to contribute more than others -- but everyone benefits.
 
-
-
 In neuroscience, data such as magnetic resonance images and physiology readings can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but incompatible specifications cause portability issues between different labs and scientists. The brain imaging data standard (BIDS) is a framework for organizing data that standardizes datasets across different researchers to enable easier sharing.
 
-## Our role
+
+## My role
```

Comment:
@Park-Patrick - I think this needs to be more of **your** contribution rather than **our** contributions. I totally love love the sentiment of us all being a team, but this is your final report and its important to highlight your unique work.

Reply:
Right makes sense! Added a few extra sentences